### PR TITLE
Add folders and components config

### PR DESCRIPTION
**Pull request by elassis**
# What I have accomplished 🏁 
- Add React files and config ✔️ 
- Create Components ✔️ 
- Add Routers and Links components ✔️ 
- Add Linters configuration Files ✔️ 
## Built With ⚒️ 
- React JS

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  "dependencies": {
+    "@testing-library/jest-dom": "^5.14.1",
+    "@testing-library/react": "^11.2.7",
+    "@testing-library/user-event": "^12.8.3",
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2",
+    "react-router-dom": "^5.3.0",
+    "react-scripts": "4.0.3",
+    "web-vitals": "^1.1.2"
+  },
```
Comment (vichuge) [inline package.json]: - [ ] It looks like you don't have react redux on your project, please install it. 🐝

File: src/components/Books.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+const Books = () => (
+  <h1>Books Page</h1>
+);
+
+export default Books;
```
Comment (vichuge) [inline src/components/Books.js]: - [ ] It looks like we can work more on book.js: 🐱 

- Takes the book data as a prop and renders the title and the author
- Has a button for removing books

`the button wouldn't remove the books yet, this is just a placeholder for now`

**Review by vichuge — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi dear student, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 
You used proper GitHub Flow ✔️
Your linter checks are all passing ✔️
Your PR messages are descriptive ✔️

## Aspects to improve ♻️

- [ ] Check the comments under the review.

- [ ] I'm afraid I couldn't find a addBooks component, please, add one following the next:
* Renders a form with inputs for the title and the author of the book
* Has a button to submit
  * the button wouldn't add the book *yet*, this is just a placeholder for now

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mikenath223 — CHANGES_REQUESTED** [review]
Hi there, 

Awesome job on making the changes requested by the previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet. Let's work on those.


 ## Required Changes ♻️
_Kindly check the comments under the review._


Happy coding, Cheers!🚀🚀

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, whether it's valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/redux/books/Books.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+const Books = () => (
+
+  <form>
+    <input type="text" placeholder="Book Title" />
+    <input type="text" placeholder="Book Author" />
+    <button type="button">Add Book</button>
+  </form>
+
+);
+
```
Comment (mikenath223) [inline src/redux/books/Books.js]: - [ ] As requested in the previous review this should take the Book title and author as props and render them. 
- [ ] It should also contain the remove book button that would just be a placeholder for now.
- [ ] In addition, please return this component to the components folder, the redux folder should only contain the redux store actions and reducers.

**Comment by elassis** [comment]
> Hi there,
> 
> Awesome job on making the changes requested by the previous reviewer 👏
> You've done well implementing some of the requested changes, but there are still some which aren't addressed yet. Let's work on those.
> 
> ## Required Changes ♻️
> _Kindly check the comments under the review._
> 
> Happy coding, Cheers!🚀🚀
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, whether it's valid or invalid unless it is requested otherwise._**
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/redux/books/Books.js [hunk 4]
```diff
@@ -1,13 +0,0 @@
-import React from 'react';
-
-const Books = () => (
-
-  <form>
-    <input type="text" placeholder="Book Title" />
-    <input type="text" placeholder="Book Author" />
-    <button type="button">Add Book</button>
-  </form>
```
Comment (vichuge) [inline src/redux/books/Books.js]: - [ ] It looks like you delete this <form> inside redux, but I'm afraid I couldn't find it in the application. Please add it. 🐶

File: src/components/BookList.js [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+      {
+      mockBooks.map((book) => (
+        <Books
+          key={book.id}
+          title={book.title}
+          author={book.author}
+        />
+      ))
+    }
+    </ul>
```
Comment (vichuge) [inline src/components/BookList.js:28]: - Hint: The bookList component is the only component inside the main page, and this doesn't have the form component. Now, imagine that the form is inside the Book component, creating a form for every book, so it couldn't be there.

**Review by vichuge — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ♻️ ⚠️ ⚠️ 🚦 🚦 

Hi dear student, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but some are still not addressed yet.

star_wars_almost_there


## Aspects to improve ♻️
_Check the comments under the review._

- [ ] Check the comments under the review. 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/BookList.js (hunk 5 above)
Comment (elassis) [reply src/components/BookList.js:28]: I already add the form to the main page but my dashboard says my project is incomplete and I don't have any reviews left... what should i do ?

**Pull request closed without merging**